include/seqan/graph_types/graph_impl_directed.h

```diff
@@ -130,23 +195,89 @@ _getVertexIdManager(Graph<Directed<TCargo, TSpec> > const& g) {
 
 //////////////////////////////////////////////////////////////////////////////
 
-template<typename TCargo, typename TSpec>
-inline typename EdgeIdHandler<Graph<Directed<TCargo, TSpec> > >::Type&
-_getEdgeIdManager(Graph<Directed<TCargo, TSpec> > const& g) {
+template <typename TCargo, typename TSpec>
+inline typename EdgeIdHandler<Graph<Directed<TCargo, TSpec> > >::Type &
+_getEdgeIdManager(Graph<Directed<TCargo, TSpec> > const & g)
+{
     SEQAN_CHECKPOINT
     typedef Graph<Directed<TCargo, TSpec> > TGraph;
     typedef typename EdgeIdHandler<TGraph>::Type TEdgeIdManager;
     return const_cast<TEdgeIdManager&>(g.data_id_managerE);
 }
 
+//////////////////////////////////////////////////////////////////////////////
+
+template <typename TCargo>
+inline void
+_copyGraph(Graph<Directed<TCargo, WithSourceId> > & dest,
+           Graph<Directed<TCargo, WithSourceId> > const & source,
+           bool const & transpose)
+{
+    SEQAN_CHECKPOINT
+    typedef Graph<Directed<TCargo, WithSourceId> > TGraph;
+    typedef typename VertexDescriptor<TGraph>::Type TVertexDescriptor;
+    typedef typename EdgeDescriptor<TGraph>::Type TEdgeDescriptor;
+    typedef typename EdgeType<TGraph>::Type TEdgeStump;
+    typedef typename Iterator<String<TEdgeStump*> const, Standard>::Type TIterConst;
+    typedef typename Iterator<String<TEdgeStump*>, Standard>::Type TIter;
+    clear(dest);
+    resize(dest.data_vertex, length(_getVertexString(source)));
+    resize(dest.data_vertex_in, length(_getVertexInString(source)));
+    TIter itInit = begin(dest.data_vertex, Standard());
+    TIter itInitEnd = end(dest.data_vertex, Standard());
+    for (;itInit != itInitEnd; ++itInit)
+        *itInit = (TEdgeStump*) 0;
+
+    TIter itInitIn = begin(dest.data_vertex_in, Standard());
+    TIter itInitInEnd = end(dest.data_vertex_in, Standard());
+    for (;itInitIn != itInitInEnd; ++itInitIn)
+        *itInitIn = (TEdgeStump*) 0;
+
+    TIterConst it = begin(source.data_vertex, Standard());
+    TIterConst itEnd = end(source.data_vertex, Standard());
+    TVertexDescriptor pos = 0;
+    for (;it != itEnd; ++it, ++pos)
+    {
+        TEdgeStump* current = *it;
+        TVertexDescriptor sourceVertex = pos;
+        while (current != (TEdgeStump*) 0)
+        {
+            TVertexDescriptor targetVertex = current->data_target;
+            // Create missing vertices
+            if (sourceVertex>targetVertex)
+            {
+                _createVertices(dest,sourceVertex);
+            }
+            else
+            {
+                _createVertices(dest,targetVertex);
+            }
```

Comment:
Please, remove the curly brackets for simple if-else statements with only one statement, as long as the scope is clear from the context (no nested if statements, all if-else if-else clauses capture only one line)